pipelines/folioRancher/folioScheduledTesting/runNightlyCypressTests/Jenkinsfile

```diff
@@ -2,7 +2,8 @@
 import org.folio.models.RancherNamespace
 import org.jenkinsci.plugins.workflow.libs.Library
 
-@Library('pipelines-shared-library@RANCHER-741-Jenkins-Enhancements') _
+@Library('pipelines-shared-library@RANCHER-1276') _
```

Comment:
Please change branch before merge

Reply:
Already have done in https://github.com/folio-org/pipelines-shared-library/pull/519